alvr/server_core/src/lib.rs

```diff
@@ -243,6 +245,57 @@ impl ServerCoreContext {
         }));
     }
 
+    pub fn get_device_motion(
+        &self,
+        device_id: u64,
+        sample_timestamp: Duration,
+    ) -> Option<DeviceMotion> {
+        dbg_server_core!("get_device_motion");
```

Comment:
Right i forget about that. The reason i would just put a line like that is because often to find a crash we don't need to know the exact moment when it happened but just the place in the codebase. I'll add more info anyway